amanuensis/blueprints/admin.py

```diff
@@ -172,6 +173,31 @@ def get_search_by_user_id():
 
     return jsonify({"filter_sets": filter_sets})
 
+@blueprint.route("/screen-institution", methods=["GET"])
+@check_arborist_auth(resource="/services/amanuensis", method = "*")
+def screen_institution():
+    name = request.args.get('name', default = "")
+    if(name == ""):
+        raise UserError("Name of institution is needed in the name argument in the url")
+    res = api_request(name)
+    total = -1
+    try:
+        total = int(res["total"])
+    except:
+        raise APIError("Possible change to or error with CSL api, see https://developer.trade.gov/api-details#api=consolidated-screening-list")
+    if(total <= 0):
+        raise APIError("Institution not found in the API, double-check spelling")
+    if(total >= 10):
+        print("The API only returns 10 results at a time, but more results match the search. If searching for one particular institution you may need to be more specific about the name")
+    try:
+        first_result_dict = res["results"][0]
+        first_id = first_result_dict["id"]
+        first_name = first_result_dict["name"]
+    except:
+        raise APIError("Possible change to or error with CSL api, unable to access required fields. See https://developer.trade.gov/api-details#api=consolidated-screening-list")
```

Comment:
What is this for?

Reply:
These were a couple of checks to make sure the api hasn't changed the format or something, is that necessary or can I get rid of it?